core/src/tasks/record/makeReady.ts

```diff
@@ -17,12 +17,16 @@ export class GrouparooRecordsMakeExports extends CLSTask {
       ? process.env.GROUPAROO_DISABLE_EXPORTS !== "true"
       : true;
 
-    const records = await RecordOps.makeExports(limit);
+    const partialRecords = await RecordOps.makeReady(limit);
 
-    if (toExport) {
-      for (const record of records) {
-        await CLS.enqueueTask("record:export", { recordId: record.id });
-      }
-    }
+    await RecordOps.makeExports(
+      partialRecords.map((r) => r.id),
+      toExport
+    );
+
+    // re-enqueue if there is more to do
+    if (partialRecords.length > 0) await CLS.enqueueTask(this.name, {});
```

Comment:
So skip waiting for the next task in ~10s, we can immediately enqueue a new instance of the task if we got records